examples/custom.ad.example.json

```diff
@@ -0,0 +1 @@
+{"1":{"src":"https:\/\/bachtrack.com\/files\/46901-banner-728x90px-uk-2016-5sec.gif","href":"https:\/\/bachtrack.com","info":"Info1"},"2":{"src":"https:\/\/bachtrack.com\/files\/42327-membership-200x200.gif","href":"http:\/\/onestoparts.com","info":"Info2"}}
```

Comment:
Done. Your comment probably got lost in the renaming to "custom" - github isn't perfect!